dashboard/db/migrate/20160706041025_add_started_to_plc_course_unit.rb

```diff
@@ -0,0 +1,7 @@
+class AddStartedToPlcCourseUnit < ActiveRecord::Migration
+  def change
+    add_column :plc_course_units, :started, :boolean
```

Comment:
should this have `default: false, null: false` or is there value in allowing null?